src/main/com/example011/radek/v1/Mines.java

```diff
@@ -0,0 +1,138 @@
+package main.com.example011.radek.v1;
```

Comment:
jinak ty metody nektere jsou porad moc dlouhe, pak veci by se tam dalo vylepsit, ale uz te nebudu dal trapit, udelej posledni pripominky a mergnu to

Reply:
Můžeme se potom k tomu klidně vrátit v nějaké další verzi